Dockerfile.konflux

```diff
@@ -0,0 +1,9 @@
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_9_1.21 AS builder
+WORKDIR /go/src/github.com/stolostron/submariner-addon
+COPY . .
+ENV GO_PACKAGE github.com/stolostron/submariner-addon
+RUN make build --warn-undefined-variables
+
+FROM registry.access.redhat.com/ubi9/ubi-minimal:latest
```

Comment:
This can be addressed later, but IIUC ACM projects are supposed to use ELS as a base image, not UBI.

Reply:
Followed up with @skitt and this is not something that's ready for us to make a change yet.